lucien-ios/Add New Comic Book/AddComicViewController.xib

```diff
@@ -0,0 +1,408 @@
+                <outlet property="coverPhotoLabel" destination="ZeW-7M-5vu" id="Fqp-sQ-Q6j"/>
+                <outlet property="datePicker" destination="Ayn-n0-Cph" id="z4L-pM-ub9"/>
+                <outlet property="genreBottomLine" destination="UCL-mM-64f" id="xbj-mc-XlW"/>
+                <outlet property="genreLabel" destination="uFa-hA-XU1" id="rjc-Tc-A75"/>
+                <outlet property="genrePicker" destination="fCM-4s-5OC" id="EEm-pY-bqH"/>
+                <outlet property="issueLabel" destination="tQp-yn-pha" id="cng-hM-SJz"/>
+                <outlet property="issueTextField" destination="Kwp-Vh-ark" id="dON-bl-xbJ"/>
+                <outlet property="publisherLabel" destination="rah-tt-duF" id="btX-ov-Iov"/>
+                <outlet property="publisherTextField" destination="uhS-Ot-Cen" id="NDF-1P-p2k"/>
+                <outlet property="releaseDateBottomLine" destination="5JR-bX-0to" id="6sO-wF-sie"/>
+                <outlet property="releaseDateLabel" destination="oyx-gl-cmU" id="1q8-pU-9Jc"/>
+                <outlet property="scrollView" destination="8pQ-9n-KZP" id="P3h-s3-pwy"/>
+                <outlet property="selectAConditionButton" destination="ZXp-2w-6eh" id="GBp-m0-CUU"/>
+                <outlet property="selectADateButton" destination="UOn-Nc-ucF" id="nyb-RN-gNw"/>
+                <outlet property="selectAGenreButton" destination="1aa-2q-qU9" id="7Jh-hk-Bo9"/>
+                <outlet property="storyTitleLabel" destination="9jV-Sr-z48" id="xJt-v8-aUR"/>
+                <outlet property="storyTitleTextField" destination="MqB-MZ-awC" id="m7w-F6-Q9E"/>
+                <outlet property="view" destination="i5M-Pr-FkT" id="sfx-zR-JGt"/>
+                <outlet property="volumeLabel" destination="xfL-O8-HUU" id="NXm-Hz-dtm"/>
+                <outlet property="volumeTextField" destination="0uv-dQ-6SL" id="yC0-hS-idC"/>
+            </connections>
+        </placeholder>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view clearsContextBeforeDrawing="NO" contentMode="scaleToFill" id="i5M-Pr-FkT">
+            <rect key="frame" x="0.0" y="0.0" width="375" height="1677"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="8pQ-9n-KZP">
+                    <rect key="frame" x="0.0" y="0.0" width="375" height="1677"/>
+                    <subviews>
+                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="COVER PHOTO" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ZeW-7M-5vu">
```

Comment:
You have constraint conflicts in this file. They need to be fixed before putting up a PR